src/com/theironyard/novauc/Main.java

```diff
@@ -4,5 +4,22 @@
 
     public static void main(String[] args) {
 	// write your code here
+
+        Student me = new Student();
+
+        int number = 9;
+        if (number != 10){
+            System.out.println("true");
+        }
+        else { System.out.println("not valid");}
+        {
+            Children me = new Children();
+            me.getName();
+            me.getAge();
+
```

Comment:
Olympew where are your constraints?  Everything else looks great.  Please add those to your project and resubmit it.